skimage/morphology/tests/test_convex_hull.py

```diff
@@ -26,6 +26,26 @@ def test_basic():
     assert_array_equal(convex_hull_image(image), expected)
 
 
+def test_empty_image():
+    image = np.array(
```

Comment:
`image = np.zeros((6, 6), dtype=bool)`

Reply:
I had just copied the format before but zeros is easier